config/locales/en.yml

```diff
@@ -180,6 +181,11 @@ en:
     modify_immutable_remote_error: 'You cannot make changes to an imported repository.'
     modify_immutable_error: 'You cannot make changes to a frozen version.'
 
+  index-table-columns:
+    assay_type_uri: 'Assay Type'
+    technology_type_uri: 'Technology Type'
+    doi: 'DOI'
+
```

Comment:
Is it worth using the Rails activemodel/record conventions to do this?

https://github.com/seek4science/seek/blob/054ef0f1646780c7e3ed47b55d3f15cb2b2b6753/config/locales/en.yml#L140-L143

https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

Reply:
I did wonder that after I'd changed it, but wasn't sure it would give any real advantage. But I wasn't aware it was already being used elsewhere so will update it to be consistent